source/includes/extracts-driver-compatibility-matrix.yaml

```diff
@@ -196,6 +196,8 @@ post: |
    In the table below, `mongodb <https://pecl.php.net/package/mongodb>`_ and
    `mongo <https://pecl.php.net/package/mongo>`_ refer to the new and legacy
    {{driver}}, respectively.
+   `PHPLIB <https://github.com/mongodb/mongo-php-library>`_ refers to the
+   userland library.
```

Comment:
This makes the `post` for `php-driver-compatibility-matrix-language` identical to the one for `php-driver-compatibility-matrix-mongodb`. If you have a better way to reduce the duplication and have them share one snippet, that'd be great.